### ISSUE TITLE
fetch failed with NetworkError for several exchanges

### ISSUE THREAD

**Issue by mikolyk** [ccxt/ccxt#18267]
### Operating System

MacOS v12.5

### Programming Languages

JS

### CCXT Version

3.1.40

### Description

These exchanges - binance, gateio,

### Return NetworkError:
```
gateio fetchTicker failed due to a network error: gateio 
GET https://api.gateio.ws/api/v4/spot/currencies fetch failed
//----
binance fetchTicker failed due to a network error: binance 
GET https://dapi.binance.com/dapi/v1/exchangeInfo fetch failed
```

I have tried to: change IP / ccxt 3.1.44 / another PC / ccxt and ccxt.pro. The error is the same. 

These exchanges working fine - okex, bybit, bitfinex, huobi

### Code

```
  const ccxt = require('ccxt');
  const exchange = new ccxt.gateio();

  try {
    await exchange.loadMarkets();

    console.log('Markets:', exchange.markets['BTC/USDT']['info']);
  } catch (e) {
    if (e instanceof ccxt.NetworkError) {
      console.log(exchange.id, 'fetchTicker failed due to a network error:', e.message);
      // retry or whatever
    } else if (e instanceof ccxt.ExchangeError) {
      console.log(exchange.id, 'fetchTicker failed due to exchange error:', e.message);
      // retry or whatever
    } else {
      console.log(exchange.id, 'fetchTicker failed with:', e.message);
      // retry or whatever
    }
  }
```

**Comment by sc0Vu** [comment]
Hi @mikolyk 

I can fetch data from those apis, would you like to check your network settings? Or add proxy?

**Comment by mikolyk** [comment]
> I have tried to: change IP / ccxt 3.1.44 / another PC / ccxt and ccxt.pro. The error is the same.

I did this. 
Also, Every thing works for other exchanges. Not working only binance and gateio

**Comment by mikolyk** [comment]
> Hi @mikolyk
> 
> I can fetch data from those apis, would you like to check your network settings? Or add proxy?

Can you give some ideas what i can check in network settings. 
 - Use proxy -  not working 
 - use 2 another servers (1 - not working, 2 - working)
 
 not working only binance and gateio, all others exchangers works perfect

**Comment by mikolyk** [comment]
**Problem is fixed, by downgrade node js to v18.16.1**
